changelog.json

```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "1.6.3-beta1": [
+      "[Fixed] Context menu doesn't open when right clicking on the edges of files in Changes list - #6296. Thanks @JQuinnie!",
+      "[Improved] Enable Git protocol v2 for fetch/push/pull operations - #6142",
+      "[Improved] Upgrade to Electron v3"
```

Comment:
thank to team support i am the how regional master creator v 1 runytry1 pajenterprises